test/integration/generated/bodyComplexRest/src/index.ts

```diff
@@ -1,12 +1,10 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import BodyComplexRestClient from "./bodyComplexRestClient";
-
 export * from "./bodyComplexRestClient";
 export * from "./parameters";
 export * from "./responses";
+export * from "./clientDefinitions";
 export * from "./models";
 export * from "./outputModels";
-
-export default BodyComplexRestClient;
+export { BodyComplexRestClient } from "./bodyComplexRestClient";
```

Comment:
If we are generating a single client, we should keep the defatult export. Named exports should only happen when generating a multi client package